src/main/java/com/example/training/service/AccidentsService.java

```diff
@@ -19,28 +20,18 @@ public AccidentsService(AccidentsProxyService accidentsProxyService) {
     }
 
     public List<AccidentDto> getAccidentsByPolicy(String policyId) {
-        var accidents = accidentsProxyService.getAccidentsByPolicy(policyId);
-        return mapToAccidentDto(accidents);
+        Accident[] accidents = accidentsProxyService.getAccidentsByPolicy(policyId);
+        List<Accident> accidentList = Arrays.asList(accidents);
```

Comment:
If you need a list instead of an array, have you tried defining the client that way? I'm not fully sure but thinking about jackson, it can map to an Array or List without a hassle. It would surprise me if Feign can't do that

Reply:
I tried to change it to an Array but wasn't possible, I had to change the AccidentMapperImpl file and I think that's not a good thing to do. Or maybe I didn't know how to do it.

Reply:
I meant to change your client to return a List instead of an array as you are moving a list around